histomicstk/segmentation/label/remove_overlap_nuclei.py

```diff
@@ -0,0 +1,58 @@
+import os
+from concurrent.futures import ProcessPoolExecutor
+
+import shapely
+from shapely.geometry import Polygon
+
+
+def create_polygon(coordinates):
+    """
+    Create a shapely Polygon from a list of points.
+
+    Args:
+        points (list): A list containing tuples representing the points of the polygon.
+
+    Returns:
+        shapely.geometry.Polygon: The polygon created from the points.
+    """
+    return Polygon(coordinates).buffer(0)
```

Comment:
The `buffer(0)` is unnecessary, at which point we may want to skip having this a separate function.

Reply:
I initially tried without `buffer(0)` but there are some self-intersecting polygons when trying this on a WSI. It throws an error with the polygon intersection part of the code if buffer(0) is not used.

Reply:
here is a sample error message when not using the `.buffer` function. `shapely.errors.GEOSException: TopologyException: side location conflict at 3530.8899999999999 267.79000000000002 0. This can occur if the input geometry is invalid.`.  The invalid geometry is created when we use self-intersecting points to generate a polygon. `.buffer(0)` removes any self-intersecting geometry.